util/feeding.py

```diff
@@ -61,28 +62,54 @@ def samples_to_mfccs(samples, sample_rate, train_phase=False):
     mfccs = contrib_audio.mfcc(spectrogram, sample_rate, dct_coefficient_count=Config.n_input)
     mfccs = tf.reshape(mfccs, [-1, Config.n_input])
 
-    return mfccs, tf.shape(input=mfccs)[0]
+    review_audio = samples
+    if FLAGS.augmentation_review_audio_steps and train_phase and any([
+                FLAGS.augmentation_spec_dropout_keeprate < 1,
+                FLAGS.augmentation_freq_and_time_masking,
+                FLAGS.augmentation_pitch_and_tempo_scaling,
+                FLAGS.augmentation_speed_up_std > 0]):
+        review_audio = gla(spectrogram)
 
+    return mfccs, tf.shape(input=mfccs)[0], review_audio
```

Comment:
The extra return value will throw an `too many values to unpack` error in `DeepSpeech.py` in `def create_inference_graph`

Reply:
fixed